### ISSUE TITLE
No 'Next' Option on Drawing Widget

### ISSUE THREAD

**Issue by henryrossiter** [ChildMindInstitute/mindlogger-app#663]
When using the Drawing widget, I am not able to click 'Next' to advance to the next item after drawing an image. The only option I have is to 'Skip'.
![image](https://user-images.githubusercontent.com/31543235/72378247-3e9f1f80-36ce-11ea-82f4-dd9e5a44412b.png)

**Comment by binarybottle** [comment]
I thought that the idea is that one should configure in the admin panel whether users can Skip a given screen or not.  If they're not allowed to Skip that screen, then there should be no button on the lower right until after they've responded, and then a Next button should appear.  If they are allowed to Skip that screen, then the Skip button appears in the lower right until they respond, and then a Next button appears.

**Comment by henryrossiter** [comment]
Yes. The issue is that even after I've responded, there is no 'Next' button.
